app/presenters/users_presenter.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+require "#{Rails.root}/lib/word_count"
+
+#= Presenter for users view
+class UsersPresenter
+  def initialize(user:)
+    @user = user
+  end
+
+  def individual_courses
+    @user.courses.where('courses_users.role = 0')
+  end
+
+  def course_string_prefix
+    Features.default_course_string_prefix
+  end
+
+  def individual_word_count
+    WordCount.from_characters CoursesUsers.where(user_id: @user.id).sum(:character_sum_ms)
+  end
+
+  def individual_upload_count
+    upload_count = 0
+    individual_courses.each do |c|
+      upload_count += c.uploads.where(user_id: @user.id).count
+    end
+    upload_count
+  end
+
+  def individual_upload_usage_count
+    upload_usage_count = 0
+    individual_courses.each do |c|
+      upload_usage_count += c.uploads.where(user_id: @user.id).sum(:usage_count)
+    end
+    upload_usage_count
+  end
+
+  def individual_article_count
+    article_ids = 0
+    individual_courses.each do |c|
+      article_ids += c.all_revisions.where(user_id: @user.id).pluck(:article_id).uniq.count
```

Comment:
if this is going to collect article_ids, and not just count them, then you don't need `count`. If you want this method to count them (as the current name indicates) then you should count them at the end, after collecting and deduplicating them.